fuzzers/frida_libpng/harness.cc

```diff
@@ -96,19 +96,32 @@ void func3( char * alloc) {
 __attribute__((noinline))
 void func2() {
   allocation = (char*)malloc(0xff);
-  printf("func2\n");
+  //printf("func2\n");
   func3(allocation);
 }
 __attribute__((noinline))
 void func1() {
-  printf("func1\n");
+    //printf("func1\n");
   func2();
 }
 // Entry point for LibFuzzer.
 // Roughly follows the libpng book example:
 // http://www.libpng.org/pub/png/book/chapter13.html
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
-  if (size < kPngHeaderSize) {
+
+   if(data[10] == 0xAB &&
```

Comment:
This test cannot be passed as now there isn't a stage using CmpLog in the lib

Reply:
should i delete it entirely or keep it as a comment?